manifests/server.pp

```diff
@@ -89,4 +94,19 @@
     class { '::mongodb::server::install': }->
     anchor { 'mongodb::server::end': }
   }
+
+  if $create_admin {
+    mongodb_user { $admin_username:
+      ensure        => present,
+      username      => $admin_username,
+      password_hash => mongodb_password($admin_username, $admin_password),
+      database      => 'admin',
+      roles         => ['dbAdmin', 'dbOwner', 'userAdmin', 'userAdminAnyDatabase'],
```

Comment:
Hello, 
It seems that 'root' role is need to pass commands for replica config:
http://docs.mongodb.org/manual/reference/built-in-roles/#root